forte/helpers/disk_io.cc

```diff
@@ -40,8 +40,10 @@
 #include "psi4/libpsio/psio.hpp"
 #include "psi4/libpsio/psio.h"
 
+#include "helpers/json.hpp"
```

Comment:
I am not sure you noticed it, json.hpp is already in the folder: `/Users/fevange/Source/forte/forte/lib/json`. Can you please update that file and point this include there?

Reply:
That folder is not in the master branch. Do you want me to move the json.hpp file to `forte/forte/lib/json`?

Reply:
The whole library might be too much for my task. I delete `json.hpp` and implement a custom version of writing occupation numbers to disk.